ios/FluentUI/People Picker/ContactView.swift

```diff
@@ -9,23 +9,22 @@ import UIKit
 
 public extension Colors {
     struct Contact {
-        public static var title: UIColor = textPrimary
-        public static var subtitle: UIColor = textSecondary
+        public static let title: UIColor = textPrimary
+        public static let subtitle: UIColor = textSecondary
```

Comment:
nit: don't need to create a second constant for textPrimary and textSecondary. Just use those inline directly.
Color constants would be useful if you declared colors using random RGB values.

Reply:
Got it. I don't think I was the one who created the color variables here. I was just changing them to be constants rather than variables since their values never changed. I believe it was part of the effort of moving all the color definitions of components out of `Colors.swift` and into their individual files. Nevertheless I can use the values inline directly as you suggested.